src/layouts/Layout.astro

```diff
@@ -0,0 +1,173 @@
+---
+import '@fontsource/roboto';
+import '@fontsource/poppins';
+
+export interface Props {
+	title: string;
+}
+
+const { title } = Astro.props;
+---
+
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="description" content="Astro description">
+		<meta name="viewport" content="width=device-width" />
+		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+		<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
+		<style>
+			@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+		  </style>
+
+		<meta name="generator" content={Astro.generator} />
+		<title>{title}</title>
+	</head>
+	<body>
+
+		<div class="banner">Learn more about our humble beginnings on the v1 version!</div>
+
+		<nav>
+			<div class="logo">
```

Comment:
I don't think you need to wrap this in a `div`. I would simply add the `logo` class to the `<a>`, and then we can change it if it turns out to be necessary